Content/posts/launching-swift-package-collections.md

```diff
@@ -0,0 +1,24 @@
+---
+date: 2021-06-09 12:00
+title: Launching Swift Package Collections
+description: What are package collections? JSON descriptions of Swift package metadata. What does the Swift Package Index have in droves? Metadata about Swift packages! As soon as we heard about package collections in Swift 5.5, we knew we had to support it.
+---
+
+It's probably not news that [this is a big week](https://developer.apple.com/wwdc21/) for Swift developers! On Monday, Apple kicked WWDC off by releasing the first beta of Xcode 13, including Swift 5.5. We've been following the release of the Swift Package Manager in Swift 5.5 closely as it implements package collections, a significant new feature.
+
+What are package collections? JSON descriptions of Swift package metadata. What does the Swift Package Index have in droves? Metadata about Swift packages! As soon as we heard about [the proposal](https://forums.swift.org/t/se-0291-package-collections/41905), we knew we had to support it. For a comprehensive description of the feature, [check out the post over on the official Swift blog](https://swift.org/blog/package-collections/).
+
+We have big plans for package collections, but we wanted to ship something alongside this first beta release. We chose to implement owner collections, a list of all packages from the index written by a repository owner. So, you could get a package collection containing [all packages from Alamofire](https://swiftpackageindex.com/Alamofire) or [all packages from Point-Free](https://swiftpackageindex.com/pointfreeco).
```

Comment:
That second sentence sounds a bit awkward. Maybe:

> So we implemented owner [author?] collections, which bundles an author’s or organisation’s complete set of packages into a collection.

(I’ll hit approve in a sec, up to you if you fancy amending!)

Reply:
Oops, double “So” opening with that.

Reply:
Went with:

> We chose to implement author collections, a list of all packages from the index grouped by repository owner.